scripts/dts-boot

```diff
@@ -5,15 +5,32 @@
 # SPDX-License-Identifier: Apache-2.0
 
 SBIN_DIR="/usr/sbin"
-BIN_DIR="/usr/bin"
 FUM_EFIVAR="/sys/firmware/efi/efivars/FirmwareUpdateMode-d15b327e-ff2d-4fc1-abf6-c12bd08c1359"
 
 export DTS_FUNCS="$SBIN_DIR/dts-functions.sh"
 export DTS_ENV="$SBIN_DIR/dts-environment.sh"
 export DTS_SUBS="$SBIN_DIR/dts-subscription.sh"
 export DTS_HAL="$SBIN_DIR/dts-hal.sh"
 export DTS_MOCK_COMMON="$SBIN_DIR/common-mock-func.sh"
-export DTS_LOG_FILE="/tmp/dts.log"
+export BASH_ENV="$SBIN_DIR/logging"
+export ERR_LOG_FILE_REALPATH
+export DTS_LOG_FILE
+export DTS_VERBOSE_LOG_FILE
+export ERR_LOG_FILE
+export SHELLOPTS
+
+ERR_LOG_FILE_REALPATH="/var/local/dts-err_$(basename "$(tty)").log"
+DTS_LOG_FILE="/tmp/dts_$(basename "$(tty)").log"
+DTS_VERBOSE_LOG_FILE="/tmp/dts-verbose_$(basename "$(tty)").log"
```

Comment:
you could also explain why you have added `ERR_LOG_FILE_REALPATH` variable, as it is not clear at first glance.

Reply:
56fd2d6b776299742a5a44341d16cb2fe89b5505

Not sure about `BASH_ENV` as it's sourced by every script we call. I would rather not assign anything else there as if for some reason we wanted to change variable value during runtime then it would only persist until we enter another script.
I mostly used it to set PS4 to be the same everywhere otherwise it's not set (even if exported) in subscripts called as command e.g. `$SBIN_DIR/dts`